scripts/update-apk.sh

```diff
@@ -69,6 +69,7 @@ if $IS_PUBLISH_BRANCH ;then
     if $PR_FOR_RELEASE ;then
         exit 0
     fi
+    cd apk
```

Comment:
Can't we just remove `cd ..` above and change paths accordingly

Reply:
@iamareebjamal i didn't get what you meant by changing paths accordingly

Reply:
The path of apk and fastlane json

Reply:
Oh yes, we can definitely do that. Let me update the pr